mortgages/src/.gitignore

```diff
@@ -0,0 +1 @@
+# empty
```

Comment:
@ge0ffrey They're automatically generated by Uberfire, see https://github.com/uberfire/uberfire/blob/master/uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemProvider.java#L1254

I'll figure out whether they can be safely removed.